### PR TITLE
Adding The official Raspberry Pi Handbook 2023 and Beginner's Guide 2020

### PR DESCRIPTION
**Pull request by ShivamPathak99**
## What does this PR do?
Add resource(s) |  | Improve repo

## For resources
I want to add the most recent edition of official Raspberry Pi Handbook and Beginner's Guide 4th Edition. 

### Description
Both of them consists of collection of Inspirational projects which totally belongs to DIY category and a useful guide for starting with RaspberryPi.

### Why is this valuable (or not)?
Yes, it will definitely add value to repo. These handbooks have been prepared by the makers of the MagPi that is the official Raspberry Pi magazine. Here the ideas has been conveyed in very creative way, the authors have used attractive photos and interactive design for quickly dragging the readers attention.

### How do we know it's really free?
It is clearly mentioned on the official TheMagPi Website that the soft copy of the Handbook is freely available to download.

### For book lists, is it a book? For course lists, is it a course? etc.
- [The Official Raspberry Pi Handbook 2023 (204-page PDF)](https://magpi.raspberrypi.com/books/handbook-2023)
- [Raspberry Pi Beginner's Guide 4th Edition (252-pages- PDF)](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up
- ***Please consider it for HactoberFest.***
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by ShivamPathak99 — COMMENTED** [review]
Please consider it for hacktoberfest.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2057,8 +2057,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Raspberry Pi: Measure, Record, Explore](https://leanpub.com/RPiMRE/read) - Malcolm Maclean (HTML)
 * [Raspberry Pi Users Guide (2012)](http://www.cs.unca.edu/~bruce/Fall14/360/RPiUsersGuide.pdf) - Eben Upton (PDF)
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
-
-
+* [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023)
+* [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)
 ### REBOL
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:2062]:
```suggestion


### REBOL
```

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2057,8 +2057,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Raspberry Pi: Measure, Record, Explore](https://leanpub.com/RPiMRE/read) - Malcolm Maclean (HTML)
 * [Raspberry Pi Users Guide (2012)](http://www.cs.unca.edu/~bruce/Fall14/360/RPiUsersGuide.pdf) - Eben Upton (PDF)
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
-
-
+* [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023)
+* [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:2061]: 2061 -> 2057
2060 -> 2059

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -2057,6 +2057,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Raspberry Pi: Measure, Record, Explore](https://leanpub.com/RPiMRE/read) - Malcolm Maclean (HTML)
 * [Raspberry Pi Users Guide (2012)](http://www.cs.unca.edu/~bruce/Fall14/360/RPiUsersGuide.pdf) - Eben Upton (PDF)
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
+* [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:2060]:
```suggestion
* [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023) (PDF)
```

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -2057,6 +2057,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
+* [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:2061]:
```suggestion
* [Raspberry Pi Beginner's Guide 4th Edition](https://magpi.raspberrypi.com/books/beginners-guide-4th-ed) (PDF)
```

**Review by shrn01 — COMMENTED** [review]
Please alphabetically sort the resources. that's why you are getting a lint error. Read docs/CONTRIBUTING.md for more informationg

**Comment by eshellman** [comment]
added in another PR

**Pull request closed without merging**